cmd/kubeadm/app/constants/constants.go

```diff
@@ -195,6 +195,9 @@ const (
 	// We need at least ten, because the DNS service is always at the tenth cluster clusterIP
 	MinimumAddressesInServiceSubnet = 10
 
+	// MaximumBitsForServiceSubnet defines maximum possible size of the service subnet in terms of bits. For example, if value is 20, then the largest supported service subnet is /12 for IPv4 and /108 for IPv6.
+	MaximumBitsForServiceSubnet = 20
```

Comment:
reading https://github.com/kubernetes/kubernetes/pull/91875#issuecomment-640614529 again.

to my understanding 108 will pass validation, but in reality anything lower than 112 will be clamped to 112 in another component. should we have a comment about that here?

Reply:
Yes, you're correct. I will add a comment here.